Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -110,6 +112,24 @@ - (dispatch_queue_t)methodQueue
     };
   }
 
+  /*
+   * The `anchor` option takes a view to set as the anchor for the share
+   * popup to point to, on iPads running iOS 8. If it is not passed, it
+   * defaults to centering the share popup on screen without any arrows.
+   */
+  if ([share respondsToSelector:@selector(popoverPresentationController)]) {
+    share.popoverPresentationController.sourceView = ctrl.view;
+    NSNumber *anchorViewTag = [RCTConvert NSNumber:options[@"anchor"]];
+    if (anchorViewTag) {
+      UIView *anchorView = [self.bridge.uiManager viewForReactTag:anchorViewTag];
```

Comment:
We should make the anchorView the sourceView in this case. Use the ctrl.view only in the else block here.